docker/dev.docker-compose.yml

```diff
@@ -0,0 +1,141 @@
+version: "2"
+services:
+  supervisord:
+    build:
+      context: ..
+      dockerfile: docker/dev.Dockerfile
+    volumes:
+      - ..:/assembl
+    command: "sh -c '. venv/bin/activate && supervisord -n'"
+    depends_on:
+      - database
+      - redis
+      - memcached
+      - elasticsearch
+    ports:
+      - "6543:6543"
+      - "8080:8080"
+  dev:
+    image: assembl_supervisor
+    volumes:
+      - ..:/assembl
+    command: "sh -c '. venv/bin/activate && supervisorctl start dev:'"
+    depends_on:
+      - supervisord
+    restart: always
+  sshd:
+    image: assembl_supervisor
+    volumes:
+      - ..:/assembl
+    hostname: cli-ssh
+    user: root
+    command: /bin/sh -c '/etc/init.d/ssh start && /usr/bin/ssh-keyscan localhost && while :; do sleep 1; done'
+  database:
+    image: postgres:9.5.5-alpine
+    volumes:
+      - ../pgdata:/var/lib/postgresql/data
+    expose:
+      - "5432"
```

Comment:
please use `127.0.0.1:5432:5432`

Reply:
pgadmin should not be on lan tho you're right